src/plugins/plugin.colors.ts

```diff
@@ -90,17 +91,19 @@ export default {
       return;
     }
 
+    const {config} = chart;
     const {
-      type,
       options: {elements},
       data: {datasets}
-    } = chart.config as ChartConfiguration;
+    } = config;
+    const type = 'type' in config ? config.type : undefined;
```

Comment:
Is this change needed? would think it'd be `undefined` in the original de-structuring style also.

Reply:
fixed